src/azure-cli/HISTORY.rst

```diff
@@ -4,7 +4,9 @@ Release History
 ===============
 
 **AppService**
+
 az webapp create : Fixing issue when running the command with --runtime 
+* functionapp: Updated container image configuration for Linux apps
```

Comment:
same comments as in another PR, history.rst is automatically generated from PR titles. Pls refine PR titles and no need to change this file.